staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -55,28 +55,17 @@ import (
 var (
 	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
 	Registry             = registered.NewOrDie("")
-	Scheme               = runtime.NewScheme()
-	Codecs               = serializer.NewCodecFactory(Scheme)
-
-	// if you modify this, make sure you update the crEncoder
-	unversionedVersion = schema.GroupVersion{Group: "", Version: "v1"}
-	unversionedTypes   = []runtime.Object{
-		&metav1.Status{},
-		&metav1.WatchEvent{},
-		&metav1.APIVersions{},
-		&metav1.APIGroupList{},
-		&metav1.APIGroup{},
-		&metav1.APIResourceList{},
-	}
+	Scheme               = scheme.Scheme
+	Codecs               = serializer.NewCodecFactory(scheme.Scheme)
```

Comment:
can we move Registry, Scheme (remove the alias here), and Codecs into the new package?

Reply:
I think we'll need the Scheme alias because we can't replace all places with `scheme.Scheme`...because that would mean using `runtime.NewScheme()` without adding the unversionedTypes, etc..